savoirfairelinux/num2words
# Fixed Hungarian NotImplementedError

## Fixes # by Gansaikhan Shur

### Changes proposed in this pull request:

* Although a support for Hungarian has been written, it would raise NotImplementedError because it was not properly initialized in __init__.py. 
* This PR fixes it and now it works for Hungarian without an issue. 

### Status

- [] WIP (Work-In-Progress)

### How to verify this change

*Fill out this section so that a reviewer can know how to verify your change.*

Please verify using
num2words(42, lang='hu')
